### DISCUSSION TITLE
Allow LLM to perform admin on remote servers.

### DISCUSSION THREAD

**Discussion by hobodrifterdavid** [open-webui/open-webui#6657] [Ideas]
Server admins need to ssh into different machines and run commands, and then follow up depending on the result of the command.

LLMs are capable of generating commands and interpreting output.

I was wonder if you would consider integrating a way to facilitate this activity (user confirms command before running, command send to remote server, a panel shows the current ssh session etc.)

**Comment by james-westbrook** [comment]
This likely wouldn't be implemented on Open WebUI itself, but could probably be implemented as a Function. It would be awesome to see someone take a shot at creating that.
